tests/SideBySide.New/BulkLoaderSync.cs

```diff
@@ -0,0 +1,498 @@
+﻿using System;
+using System.Data;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using Xunit;
+using Dapper;
+
+namespace SideBySide.New
+{
+	[Collection("BulkLoaderCollection")]
+	public class BulkLoaderSync : IClassFixture<DatabaseFixture>
+	{
+		public BulkLoaderSync(DatabaseFixture database)
+		{
+			m_database = database;
+			//xUnit runs tests in different classes in parallel, so use different table names for the different test classes
+			string testClient;
+#if BASELINE
+			testClient = "Baseline";
+#else
+			testClient = "New";
+#endif
+			m_testTable = "test.BulkLoaderSyncTest" + testClient;
+
+			m_initializeTable = @"
+				create schema if not exists test;
+				drop table if exists " + m_testTable + @";
+				CREATE TABLE " + m_testTable + @"
+				(
+					one int primary key
+					, ignore_one int
+					, two varchar(200)
+					, ignore_two varchar(200)
+					, three varchar(200)
+					, four datetime
+					, five blob
+				) CHARACTER SET = UTF8;";
+			m_removeTable = "drop table if exists " + m_testTable + @";";
+			m_memoryStreamBytes = System.Text.Encoding.UTF8.GetBytes(@"1,'two-1','three-1'
+2,'two-2','three-2'
+3,'two-3','three-3'
+4,'two-4','three-4'
+5,'two-5','three-5'
+");
+		}
+
+		[Fact]
+		public void BulkLoadTsvFile()
+		{
+			try
+			{
+				InitializeTest();
+
+				MySqlBulkLoader bl = new MySqlBulkLoader(m_database.Connection);
```

Comment:
Nitpick: prefer to use [object initialisation syntax](https://msdn.microsoft.com/en-us/library/bb397680.aspx) here.